File: iCovieApi/iCovieApi/Models/Master/BedTypeModel.cs
using System;

using System;
using Newtonsoft.Json;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace iCovieApi.Models
{
    public class BedTypeModel
    {
        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int id { get; set; }
        
        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string bedtype { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string shortname { get; set; }


        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int cid { get; set; }
    }
}

File: iCovieAdmin-Ng/New folder/src/app/core/auth/_models/Masters/bedtype.model.ts


export class BedTypeModel {
    id: number;    
    bedtype:string;   
    
    cid:number; 
    isCore: boolean = false;    
    clear(): void {
        this.id= undefined;
        this.bedtype= '';    
              
        this.cid = undefined;      
        this.isCore=false;
	}
}

